### PR TITLE
Set up the structure for the API PHP testing for TA_Feedback 

### PR DESCRIPTION
**Pull request by DarrenZhang01**
This file is a structure for the testing, and it is not complete. The reason that I make this Pull Request is that after being merged, Daniel and I can separate the testing cases in this file and work on the same file from now.

### CODE REVIEW

File: backend_testing/test.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
+// This is the mocha testing file for test.php
+import "isomorphic-fetch";
+import chai from "chai";
+
+describe("GET testing", function () {
```
Comment (F-Ting) [inline backend_testing/test.js]: Use a better description like "Test GET request to API"

File: backend_testing/test.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+import "isomorphic-fetch";
+import chai from "chai";
+
+describe("Test GET request to API", function () {
+    it("get the list of users", async function () {
+        let fetched = await fetch("localhost:3000/test.php?url=users%2F")
+        .catch(function (err) {
+            throw(err);
+        });
+        // TODO: Use the expect statement to test the fethced data
```
Comment (richardzeng1) [inline backend_testing/test.js]: Please remove or implement. Do not merge TODO comments.

**Comment by DarrenZhang01** [comment]
@david-yz-liu For 2, we will be using a "test.php" file to deal with all the API fetching and operating on the database. I can also change it if the file name changes.

**Comment by DarrenZhang01** [comment]
@david-yz-liu Thanks for pointing out the spelling error. I revised it.

**Pull request merged**